cfgov/jinja2/v1/_includes/organisms/filterable-list-controls.html

```diff
@@ -0,0 +1,174 @@
+
+   filterable_list_controls.render()
+
+   ==========================================================================
+
+   Description:
+
+   Create an expandable of filters when given:
+
+   settings:            Array of objects describing a form structure.
+                        Example nested form:
+                        {
+                            'type': 'form',
+                            'children': [
+                                {
+                                    'type': 'text',
+                                    'label': 'Topics',
+                                    'cols': '1-2',
+                                    'attrs': { 'placeholder': 'Search for topics' }
+                                },
+                                {
+                                    'type': 'text',
+                                    'label': 'Author',
+                                    'cols': '1-2',
+                                    'attrs': { 'placeholder': 'Search for authors' }
+                                },
+                                {
+                                    'type': 'form',
+                                    'label': 'Date range',
+                                    'children': [{
+                                            'type': 'text',
+                                            'label': 'From:',
+                                            'cols': '1-2',
+                                            'attrs': { 'placeholder': 'DD/MM/YYYY' }
+                                        },
+                                        {
+                                            'type': 'text',
+                                            'label': 'To:',
+                                            'cols': '1-2',
+                                            'attrs': { 'placeholder': 'DD/MM/YYYY' }
+                                        }]
+                                }]
+                        }
+
+   options:             Object of optional settings.
+   options.form_method: "post" or "get" form method.
+                        Default is "post."
+   options.action:      The form action URL.
+
+   ========================================================================== #}
+
+{% macro render(settings=[], options={}) %}
+{% import 'molecules/expandable.html' as expandable %}
+
+<div class="o-filterable-list-controls">
+    {% call expandable.render("Filter", {
+      'is_bordered': true,
+      'is_midtone': true,
+      'is_expanded': true
+    }) %}
+        {{ _filters_form(settings, options) }}
+    {% endcall %}
+</div>
+
+{% endmacro %}
+
+
+{# A helper macro for filterable_list_controls.render()
+   ========================================================================== #}
+{% macro _filters_form(inputs=[], options={}) %}
+{% if options.action %}
+{% set action = options.action if options.action != '/' %}
```

Comment:
I get `undefined value printed: the inline if-expression on line 76 in 'organisms/filterable-list-controls.html' evaluated to false and no else section was defined.`